core/conversion/converters/impl/normalize.cpp

```diff
@@ -79,37 +120,48 @@ auto normalize_registrations TORCHTRT_UNUSED =
                auto axes_values = args[1].unwrapToIntList().vec();
                auto keep_dims = args[2].unwrapToBool();
 
-               int32_t axes_mask = 0;
-               auto self_nb_dims = self->getDimensions().nbDims;
-               for (size_t i = 0UL; i < axes_values.size(); ++i) {
-                 auto axis = axes_values[i];
-                 if (axis < 0) {
-                   axis += self_nb_dims;
-                 }
-                 TORCHTRT_CHECK(
-                     axis < self_nb_dims,
-                     "aten::frobenius_norm axis: " << i << " with value: " << axis << " exceeds input rank");
-                 axes_mask += 1 << axis;
-               }
+               auto axes_mask = axes_mask_from_axes_values(n, self->getDimensions().nbDims, axes_values);
 
-               auto squared_layer = add_elementwise(
-                   ctx, nvinfer1::ElementWiseOperation::kPROD, self, self, util::node_info(n) + "_squared");
-               TORCHTRT_CHECK(squared_layer, "Unabled to create square layer from node: " << *n);
-               auto squared_output = squared_layer->getOutput(0);
-
-               auto sum_layer =
-                   ctx->net->addReduce(*squared_output, nvinfer1::ReduceOperation::kSUM, axes_mask, keep_dims);
-               TORCHTRT_CHECK(sum_layer, "Unable to create sum layer from node: " << *n);
-               sum_layer->setName((util::node_info(n) + "_sum").c_str());
-               auto sum_output = sum_layer->getOutput(0);
-
-               auto sqrt_layer = ctx->net->addUnary(*sum_output, nvinfer1::UnaryOperation::kSQRT);
-               TORCHTRT_CHECK(sqrt_layer, "Unable to create sqrt layer from node: " << *n);
-               sqrt_layer->setName((util::node_info(n) + "_sqrt").c_str());
-               auto sqrt_output = sqrt_layer->getOutput(0);
+               auto norm = frobenius_norm(ctx, n, self, axes_mask, keep_dims);
+               auto out = ctx->AssociateValueAndTensor(n->outputs()[0], norm);
+               LOG_DEBUG("Output tensor shape: " << out->getDimensions());
+               return true;
+             }})
+        .pattern(
+            {"aten::linalg_norm(Tensor self, Scalar? ord=None, int[1]? dim=None, bool keepdim=False, *, int? dtype=None) -> (Tensor)",
+             [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
+               // https://pytorch.org/docs/stable/generated/torch.linalg.norm.html
+               auto self = args[0].ITensorOrFreeze(ctx);
+               TORCHTRT_CHECK(
+                   args[1].IValue()->isNone(),
+                   "aten::linalg_norm converter does not yet support non-None 'ord' arguments.");
```

Comment:
Can we add a recommendation to add `aten::linalg_norm` to `torch_executed_ops` in this case?